objax/nn/layers.py

```diff
@@ -327,6 +327,63 @@ def __call__(self, x: JaxArray) -> JaxArray:
         self.avg.value += (self.avg.value - x) * (self.momentum - 1)
         return self.avg.value
 
+class RNN(Module):
+    """ Recurrent Neural Network (RNN) block."""
```

Comment:
```suggestion
    """Recurrent Neural Network (RNN) block."""
```
Extra whitespace in the beginning

Reply:
Done